Tools/scripts/checkpyc.py

```diff
@@ -51,7 +51,7 @@ def main():
                     print('Bad MAGIC word in ".pyc" file', end=' ')
                     print(repr(name_c))
                     continue
-                mtime = get_long(mtime_str)
+                mtime = get_long(mtime_str) & 0xFFFF_FFFF
```

Comment:
Sees the definition of get_long(). It returns an unsigned 32-bit integer, and returns -1 only on error. You need to convert not it, but the file timestamp.